src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+
+import { Link, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { SexEnum } from '../../types/SexEnum';
+import { HighlightedText } from '../HighlightedText';
+import { FilterEnum } from '../../types/FilterEnum';
+
+type Props = {
+  person: Person
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, slug, sex } = person;
+  const [searchFilters] = useSearchParams();
+  const q = searchFilters.get(FilterEnum.Query);
+
+  return (
+    <Link
+      to={{
+        pathname: `../${slug}`,
```

Comment:
The relative pathname `../${slug}` could lead to navigation errors depending on the current URL. It's better to use an absolute path.